lib/v2/screens/profile_screens/guardians/guardians_tabs/guardians_screen.dart

```diff
@@ -17,20 +24,30 @@ class GuardiansScreen extends StatelessWidget {
             listenWhen: (_, current) => current.pageCommand != null,
             listener: (context, state) {
               var pageCommand = state.pageCommand;
+              BlocProvider.of<GuardiansBloc>(context).add(ClearPageCommand());
+
               if (pageCommand is NavigateToSelectGuardians) {
                 NavigationService.of(context).navigateTo(Routes.selectGuardians, pageCommand.myGuardians);
+              } else if (pageCommand is ShowRecoveryStarted) {
+                _showRecoveryStartedBottomSheet(context, pageCommand.guardian);
+              } else if (pageCommand is ShowRemoveGuardianView) {
+                _showRemoveGuardianDialog(context, pageCommand.guardian);
+              } else if (pageCommand is ShowMessage) {
+                SnackBarInfo(title: pageCommand.message, context: context);
```

Comment:
ScaffoldMessenger.of(context).showSnackBar(
                SnackBarInfo(title: 'Do not forget the snack method', context: context),
              );

Reply:
SnackBarInfo(title: pageCommand.message, context: context).show(context);
void show(BuildContext context) {
    ScaffoldMessenger.of(context).showSnackBar(this);
  }

Reply:
next pr added this.